### ISSUE TITLE
Float value doesn't appear

### ISSUE THREAD

**Issue by mustafakoleli** [bernii/gauge.js#78]
Hi,

I use gauge.js library, i think great library, thank you for this.

But i have a problem;
I want to show float number  in gauge. Can i do it on display?

I'm sorry for my bad English :(

**Comment by ThomasCrevoisier** [comment]
Nope, the decimal part will be ignored as you can see here : https://github.com/bernii/gauge.js/blob/gh-pages/dist/gauge.coffee#L63

**Comment by mustafakoleli** [comment]
Thanks for the reply.

**Comment by subhadipghorui** [comment]
How display the decimal part og gauge ??

**Comment by subhadipghorui** [comment]
Solved in here.
https://github.com/bernii/gauge.js/issues/162
